docs/public/operator-hub.adoc

```diff
@@ -1,271 +1,151 @@
-section>>.
-
-=== [[create-service]]Create a service from an Operator 
-
-[NOTE]
-====
-For the commands mentioned in this section to work properly, you need to make
-sure that the Operator has a valid definition in its `metadata` to start the
-requested service. The commands mentioned here refer the
-`metadata.annotations.alm-examples` of an Operator and use it as-is to start
-the service. If this YAML has placeholder values or sample values that are not
-meant to aid in starting a real service, you will not be able to see a service
-start from it.
-====
-
-To start an `EtcdCluster` service from `etcdoperator.v0.9.4` Operator, you need
-to execute:
+In above output, `etcdoperator.v0.9.4` is the Operator while `EtcdCluster`,
+`EtcdBackup` and `EtcdRestore` are the CRDs provided by this Operator.
 
-[source,shell]
-----
-$ odo service create etcdoperator.v0.9.4 --crd EtcdCluster
-----
 
-This is exactly same command as that shown in <<dry-run,printing the YAML of
-service>> section above but without the `--dry-run` flag.
+=== Deploying etcd to a cluster
 
-At the moment, `odo` is unable to list services started from an Operator. To
-check if the above command succeeded in starting a service, use `kubectl` or
-`oc`:
+In this example, we will be deploying our link:https://operatorhub.io/operator/etcd[installed etcd Operator] to an OpenShift / Kubernetes cluster.
 
-[source,shell]
+. Create a etcd service from the `etcdoperator.v0.9.4` Operator:
++
+[source,sh]
 ----
-$ kubectl get EtcdCluster
+  $ odo service create etcdoperator.v0.9.4 --crd EtcdCluster
 ----
 
-At the time of writing this document, above command worked out of the box
-because, as mentioned in the note, `etcdoperator.v0.9.4` 's definition has a
-valid example for `EtcdCluster` embedded into it. This can be checked by doing:
-
-
-[source,shell]
-----
-$ kubectl get csv/etcdoperator.v0.9.4 -o yaml
+. Confirm the Operator was deployed:
++
+[source,sh]
 ----
+  # Kubernetes
+  $ kubectl get EtcdCluster
+  $ kubectl get csv/etcdoperator.v0.9.4 -o yaml
 
-and referring to the `alm-examples` section under `annotations` in the
-`metadata` of the `etcdoperator.v0.9.4` Operator.
-
-If you're using OpenShift, you can replace `kubectl` with `oc` in above
-command.
-
-If there is placeholder/invalid data or no data in the aforementioned section
-of the Operator's definition, `odo` won't be able to start the service. As an
-example, refer to the YAML definition of `EtcdBackup` in the
-`etcdoperator.v0.9.4` 's `metadata`:
-
-[source,yaml]
-----
-apiVersion: etcd.database.coreos.com/v1beta2
-kind: EtcdBackup
-metadata:
-  name: example-etcd-cluster-backup
-spec:
-  etcdEndpoints:
-  - <etcd-cluster-endpoints>
-  s3:
-    awsSecret: <aws-secret>
-    path: <full-s3-path>
-  storageType: S3
+  # OpenShift
+  $ oc get EtcdCluster
+  $ oc get csv/etcdoperator.v0.9.4 -o yaml
 ----
++
+[IMPORTANT]
+====
+The above command will be **REPLACED** with an `odo list` equivalant in the future.
```

Comment:
Actually we can use `odo service list` here since #3069 should go in pretty soon. It's approved and lgtm'd as well.

Reply:
If we don't, it's `odo service list`, not `odo list`.